modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -401,6 +436,7 @@ The allowed property types for property type constraints are:
 * `ZONED DATETIME`
 * `DURATION`
 * `POINT`
+* `VECTOR<TYPE>(DIMENSION)` label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2025.10]
```

Comment:
should we say what TYPE and DIMENSION is here? it's mentioned on the syntax page in the note below, and exists up by the example of vector property type constraints but not in this list of allowed types